public/static/locales/eng/terms.md

```diff
@@ -1,51 +1,51 @@
-# Terms and conditions
+# Terms for participation
 
 NOTE: Translation efforts are in progress. Until then, please direct any questions to samromur@ru.is.
 
-Á vefsvæðinu [samrómur.is](https://www.samromur.is) getur þú tekið þátt í gerð gagnasafnsins Samrómur. Í þátttöku þinni felst að þú skilar inn eigin raddupptöku og eftir atvikum, lýðfræðilegum upplýsingum, yfirferðum og endurgjöf sem mun nýtast við gerð opins radd gagnasafn.
+On this website you can take part in the creation of the Samrómur database. By participating, you submit your own voice recording and, as appropriate, demographic information, reviews, and feedback that will be useful in creating an open voice database.
 
-Gagnasafnið verður aðgengilegt á [www.samrómur.is](www.samromur.is). Með gerð raddgagnasafnsins verður hægt að styðja við og efla þróun máltæknilausna fyrir íslensku.
+The database will be accessible at [The Database](www.samromur.is/gangasafn). With the creation of the voice database, it will be possible to support and strengthen the development of language technology solutions for Icelandic.
```

Comment:
```suggestion
The database will be accessible at [www.samromur.is](www.samromur.is/gagnasafn). With the creation of the voice database, it will be possible to support and strengthen the development of language technology solutions for Icelandic.
```